kernel.py

```diff
@@ -105,6 +97,22 @@ def __init__(self, connection_filename="kernel.json", connection_fn_with_pid=Tru
             logger.addHandler(logging.StreamHandler(sys.stdout))
         self._logger = logger
 
+    def _get_connection_filename(self, connection_filename, connection_fn_with_pid):
+        """Craft connection file path"""
```

Comment:
There should be a newline after and before `"""`. And the parameters and the return type should be documented. I.e. like:
```
"""
:param str connection_filename:
:param bool connection_fn_with_pid:
:return: crafted connection file path
:rtype: str
"""
```

Reply:
Fix: 4914afb70bc (see below)